shared/go/pkg/valueobject/description_test.go

```diff
@@ -0,0 +1,49 @@
+/*
+ *  Copyright 2021 Data and Service Center for the Humanities - DaSCH.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package valueobject_test
+
+import (
+	"github.com/dasch-swiss/dasch-service-platform/shared/go/pkg/valueobject"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNewDescription(t *testing.T) {
```

Comment:
I think the naming convention is Test_whatever. At least, this is what IntelliJ is proposing.

Reply:
I based my method names off of the methods in `email_test.go`. Should I change them? I don't get any proposal from IntelliJ 🤔

Reply:
Ikve mixed it up in my head. It is something like this: https://github.com/dasch-swiss/dasch-service-platform/blob/main/services/admin/backend/infrastructure/repository/user/user_test.go

`TestPackage_MethodUnderTest` but it doesn't really matter. Over time, we will find our convention.